Sources/TuistDocTesting/MockSwiftDocController.swift

```diff
@@ -0,0 +1,12 @@
+import TuistDoc
+@testable import TuistCore
+
+final class MockSwiftDocController: SwiftDocControlling {
+    func generate(format: SwiftDocFormat,
```

Comment:
I'd add a stub method here (which you'll probably need in the unit tests anyway)

Reply:
989e14dd